ci-cd-bootstrap/params/mvn-build-pod/build_params

```diff
@@ -0,0 +1,5 @@
+SOURCE_REPOSITORY_URL=https://github.com/InfoSec812/labs-ci-cd
+SOURCE_CONTEXT_DIR=docker/mvn-build-pod
+BUILDER_IMAGE_NAME=registry.access.redhat.com/openshift3/jenkins-slave-maven-rhel7:latest
```

Comment:
I would recommend against using `:latest` here, and pin it to the version of Openshift you plan to use (i.e. `v3.5`,v3.6` etc.) they update these images frequently, and they are not necessarily backwards compatible.